staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1/types.go

```diff
@@ -78,8 +78,8 @@ const (
 // Event captures all the information that can be included in an API audit log.
 type Event struct {
 	metav1.TypeMeta `json:",inline"`
-	// ObjectMeta is included for interoperability with API infrastructure.
-	// +optional
+
+	// Required. The name field under metadata is necesssary.
```

Comment:
@liggitt - should these also be changed?

Reply:
> // ObjectMeta is included for interoperability with API infrastructure.

Oh my bad. Removed this comment too by mistake